docs/gds-supported-platform/getting-started-gsp.md

```diff
@@ -0,0 +1,167 @@
+# Getting started with a GDS Supported Platform cluster
+
+## Before you start
+
+You must have:
```

Comment:
Can we relax this slightly to "should have"? It isn't the case that you _must_ have a 12 factor-style app (you probably _should_ though).

Reply:
Relaxed 😎